core/src/test/resources/filter.properties

```diff
@@ -0,0 +1,2 @@
+keywords=TransactionImple,Periodic Recovery,TransactionSynchronizationRegistryImple
+
```

Comment:
That's exactly the reason why the JIRA is incomplete. A good JIRA will explain the requirement and perhaps where the requirement originates from.

Why is there not a need for filters that block lines and other filters that match lines. If there is such a need then you would need at least two property names, for example: matching_keywords and blocked_keywords